src/Symfony/Component/HttpFoundation/Session/Storage/Handler/PdoSessionHandler.php

```diff
@@ -486,10 +486,32 @@ private function buildDsnFromUrl(string $dsnOrUrl): string
             $driver = substr($driver, 4);
         }
 
+        $dsn = null;
```

Comment:
We don't need this. 🤓

Reply:
We do.. Or we will reference an undefined variable. 

```php
case 'pgsql':
  $dsn = $dsn ?? 'pgsql:';
```

Is it okey to do `??` on a variable not defined?